cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,36 @@
 describe('', () => {
   before(() => {
-
+    cy.task('generateUser').then((user) => {
+      cy.login(user.email, user.username, user.password);
+    });
   });
 
-  it('', () => {
-
+  it('create an article', () => {
+    let newArticle;
+    cy.visit('editor');
+    cy.task('generateArticle').then((article) => {
+      newArticle = article;
+      cy.get(':nth-child(1) > .form-control').type(newArticle.articleTitle);
```

Comment:
This is a badly constructed selector. Try to avoid using `>`, `:nth-child` and too many spaces.

Reply:
fixed